lib/gitlab/client/branches.rb

```diff
@@ -82,7 +88,6 @@ def create_branch(project, branch, ref)
     #
     # @param  [Integer, String] project The ID or name of a project.
     # @param  [String] branch The name of the branch to delete
-    # @return [Gitlab::ObjectifiedHash]
```

Comment:
You're right, the offset in the diff looked like it was `create_branch`.  Whoops!  I still don't think it should be removed... Can we just say it returns a `Boolean`?